src/hotspot/share/oops/instanceKlass.hpp

```diff
@@ -1080,6 +1080,12 @@ class InstanceKlass: public Klass {
   bool idnum_can_increment() const      { return has_been_redefined(); }
   inline jmethodID* methods_jmethod_ids_acquire() const;
   inline void release_set_methods_jmethod_ids(jmethodID* jmeths);
+  // Used to explicitly clear jmethodIDs for the contained methods
+  // This is required for JDK-8313816 but should not be used otherwise!
```

Comment:
This comment is not helpful - what does it actually mean? Referring to a bugid doesn't help.